modules/mobile/docs/openapi.yaml

```diff
@@ -21,6 +21,181 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Message"
+
+  /v0/appointments:
+    get:
+      summary: /v0/appointments
+      description: Given a date range returns a list of upcoming VA, Community Care, and Express Care appointments
+      operationId: getAppointments
+      security:
+        - Bearer: [ ]
+      parameters:
+        - in: query
+          name: startDate
+          schema:
+            type: datetime
+          description: The start date for the range of appointments in ISO 8601 UTC format
+          example: 2020-10-29T07:00:00.000Z
+        - in: query
+          name: endDate
+          schema:
+            type: datetime
+          description: The end date for the range of appointments in ISO 8601 UTC format
+          example: 2021-11-29T08:00:00.000Z
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Appointments"
+              examples:
+                VA Appointment:
+                  value:
+                    data:
+                      - type: appointment
+                        id: "23fe358d-6e82-4541-804c-ce7562ba28f4"
+                        attributes:
+                          appointmentType: va
+                          status: BOOKED
+                          startTime: 2020-10-29T10:30:00Z
+                          minutesDuration: 60
+                          comment: Please arrive 20 minutes before the start of your appointment
+                          timeZone: America/Los_Angeles
```

Comment:
Have we considered encoding the TZ into the `startTime`? This would become `startTime: 2020-10-29T03:30:00-07:00` (that's an early appointment)